scripts/g.extension/g.extension.py

```diff
@@ -118,9 +118,16 @@
 #% description: Operate on toolboxes instead of single modules (experimental)
 #% suppress_required: yes
 #%end
+#%flag
+#% key: p
+#% description: Download Add-Ons paths json file
```

Comment:
```suggestion
#% description: Generates JSON file containing the download URLs of the official Addons
```

Reply:
I fixed it as suggested in [6b60462](https://github.com/OSGeo/grass/commit/6b60462490db09a338839f4fc56a8ef9ca056bd5).